README.md

```diff
@@ -1 +1,164 @@
-# Stock-Metrics-WebApp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Stock-Metrics-WebApp</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Stock-Metrics-WebApp <a name="Stock-Metrics-WebApp"></a>
+
+Companies is a React and Redux SPA application that displays interesting information about various companies in the world. It fetchs all the information that it displays using API request to a server.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- React
+- Redux
+- Git, Github
+- VSCode
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Git,
+> Github,
+> React
+> Redux
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/react/">React</a></li>
+
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Basic set up
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+In order to run this project you need:
+
+1. Prerequisites
+In order to run this project you need:
+
+- Visual Studio
+- Github
+- NodeJS
+- React & Redux
+- Browser
+
+2. Setup
+Clone this repository to your desired folder:
+
+git clone https://github.com/Ashnaali3255/Stock-Metrics-WebApp.git
+
+3. Install:
+
+To install you should use this command:
+- npm install
+
+4. Run tests
+ 
+ To run tests, run the following command:
+
+ 1. For styles: npx stylelint "**/*.{css,scss}"
+ 2. For Javascript: npx eslint .
+
+ To fix it. run the following command:
+
+ 1. For styles: npx stylelint "**/*.{css,scss}" --fix
+ 2. For Javascript: npx eslint . --fix
+
+ For REACT & Redux:
+ 
+ Run this command for JS:
+
+ 1. npx eslint "**/*.{js,jsx}"
+
+ To fix use this command:
+
+ 1. npx eslint "**/*.{js,jsx}" --fix
+
+
+6. Deployment
+ 
+ You can deploy this project using:
+ "Github Pages"
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://companies-metrics-webapp.onrender.com)
+
+👤 **Ashna Ali**
```

Comment:
- [ ] Let's endeavor to be thorough in our documentation. The live demo section is wrapped into the Authors section, which is not good. Kindly remove it from here and have it as a standalone section like the others.

- [ ] You have not included the link to the presentation as requested previously. Kindly record the video and add the link to the readme file.

- [ ] **[OPTIONAL]** Most of the links in your readme file headers are broken. Fixing them would contribute to a more professional documentation.